### ISSUE TITLE
Chromecast driver - receiver not found

### ISSUE THREAD

**Issue by joeyparrish** [google/generic-webdriver-server#19]
The Chromecast driver seems to have stopped working:

```
(node:5795) UnhandledPromiseRejectionWarning: Error: Failed to launch receiver!  Reason: NOT_FOUND
    at Channel.receiver.on (/usr/local/lib/node_modules/chromecast-webdriver-server/cast-utils.js:122:18)
```

**Comment by joeyparrish** [comment]
Can't be reproduced on the Chromecast device in my home.  The device in our lab must be broken in some way.  I'll leave this open while I investigate further.

**Comment by joeyparrish** [comment]
Confirmed that the tool works (v1.0.3) with a Chromecast Ultra in my home and a Chromecast v1 in our lab network.  It's only failing on our lab's Chromecast Ultra.  There must be something wrong with the device.

The failing Ultra is responding to pings and the Cast protocol, but isn't functional.  It has firmware 1.50, though my home Ultra has firmware 1.56.  So our lab device is badly out of date.  I don't have physical access at the moment, so it's not clear what state it's in otherwise.

I tried using a tethered Android device in our lab to factory reset the Chromecast Ultra in the lab, but it rejected this command.  I suspect the device may be effectively unusable and just needs to be replaced.
